src/js/variant-widget/eva-variant-widget-panel.js

```diff
@@ -149,7 +152,7 @@ EvaVariantWidgetPanel.prototype = {
                     header: {
                         baseCls: 'eva-header-1'
                     },
-                    title: 'Variant Browser <span class="icon icon-generic title-header-icon" data-icon="i"  data-qtip="Search the EVA variant warehouse using any combination of the filtering options on the left hand-side. Search results can be exported in CSV format and individual variants can be further investigated using the in-depth Variant Data tabs found below the main results table." style="margin-bottom:0px;"></span>',
+                    title: 'Variants found <span class="icon icon-generic title-header-icon" data-icon="i"  data-qtip="Search the EVA variant warehouse using any combination of the filtering options on the left hand-side. Search results can be exported in CSV format and individual variants can be further investigated using the in-depth Variant Data tabs found below the main results table." style="margin-bottom:0px;"></span>',
```

Comment:
The `data-qtip` field is no longer necessary since everything is explained at the top paragraph.